qiskit_machine_learning/neural_networks/circuit_qnn.py

```diff
@@ -43,19 +43,21 @@ def __init__(self, circuit: QuantumCircuit,
             input_params: The parameters of the circuit corresponding to the input.
             weight_params: The parameters of the circuit corresponding to the trainable weights.
             interpret: Determines the output format, possible choices are:
-                * 'tuple' (default): a tuple of binary values, e.g. (0, 1, 0, 1, 0)
+                * 'int' (default): an integer corresponding to the bitstring, e.g. '01010' -> 10
                 * 'str': a bitstring of type str, e.g. '01010'
-                * 'int': an integer corresponding to the bitstring, e.g. 10
+                * 'tuple': a tuple of binary values, e.g. (0, 1, 0, 1, 0)
                 * a custom callable that takes a sample of type 'tuple' and maps it to some other
                 output, output should be hashable for sparse representation of probabilities
                 and probability gradients.
             dense: Whether to return a dense (array with 'output_shape') or sparse (dict)
                 probabilities. Dense probabilities require "interpret == 'int'" where the integer
                 will be the index in the array of probabilities.
```

Comment:
Code below suggests that 'callable' is also valid, given certain output from it, for dense. That would be nice to state here too